WcaOnRails/app/models/user.rb

```diff
@@ -670,6 +670,27 @@ def serializable_hash(options = nil)
     json
   end
 
+  def to_wcif(registration = nil)
+    person_pb = [person&.ranksAverage, person&.ranksSingle].compact.flatten
+    {
+      "id" => 0,
```

Comment:
Yes, and I have still no clue if the "solution" to have an empty registration for all managers is a good one..
Basically it means we would create an invalid object in the db (which would need to be filtered out when displaying to the public), and I'm not sure we want to have such thing :s